__tests__/constants.js

```diff
@@ -10,3 +10,22 @@ test('getPathKey', () => {
   expect(getPathKey('linux', {})).toBe('PATH');
   expect(getPathKey('darwin', {})).toBe('PATH');
 });
+
+test('isProduction', () => {
```

Comment:
Yah, wasn't sure of the overall test structure / style. It seemed that mosts tests are at parent level to each file, so I didn't know if nesting tests inside of a describe was good or bad style here.

> Btw. changing globals like process.env.NODE_ENV in tests is a bit dangerous since we run tests in parallel
Agreed. Wasn't sure of how to successfully mock this, but your advice above sorts that out :)